staging/src/k8s.io/apiserver/pkg/server/options/authorization.go

```diff
@@ -84,31 +95,43 @@ func (s *DelegatingAuthorizationOptions) ApplyTo(c *server.AuthorizationInfo) er
 		return nil
 	}
 
-	cfg, err := s.ToAuthorizationConfig()
-	if err != nil {
-		return err
-	}
-	authorizer, err := cfg.New()
+	authzr, err := s.ToAuthorization()
```

Comment:
nit: `authzr` -> `authorizer`. you save a few characters but it's a really awkward var name

Reply:
it shadows an import then. golint won't be happy.

Reply:
renamed to a